tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +334,47 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_multistage_dockerfile(dfobj):
+    """Given a dockerfile object, return if it is a multistage dockerfile,
```

Comment:
small nit: `return True or False if it is a multistage dockerfile`

Reply:
Also, do you see using the True/False flag? The only place where `check_multistage_dockerfile` is used in this PR, you throw away the True/False return value. Perhaps we only return the indexes of the FROM lines in the dfobj structure for this function? If the length of the list that you return is > 1, then we know it is multistage.